utils/sl/prereq_tools/base.py

```diff
@@ -557,6 +558,21 @@ def configure(self, config, prereqs):
                 return True
         return False
 
+def ensure_dir_exists(dirname, dry_run):
+    """Ensure a directory exists"""
+    my_e = IOError(errno.EEXIST, 'File exists', dirname)
+    if not os.path.exists(dirname):
+        if dry_run:
+            print("Would create %s" % dry_run)
+            return
+        try:
+            os.makedirs(dirname)
+        except Exception as e:
+            my_e = e
```

Comment:
(pylint-redefined-variable-type) Redefinition of my_e type from OSError to Exception